### PR TITLE
Closes #1 - Create index.html markup

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by strelovidniy**

### CODE REVIEW

File: intdex.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
```
Comment (RamzesUAA) [inline intdex.html:10]: Plz close body.

**Review by RamzesUAA — CHANGES_REQUESTED** [review]

File: intdex.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+</body>
+
+</html>
```
Comment (anastasiiakolomyi) [inline intdex.html:14]: Do something

**Review by IhorSolovii — APPROVED** [review]

File: intdex.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (r-kovalch) [inline intdex.html:1]: no

**Review by r-kovalch — COMMENTED** [review]
fix

**Review by r-kovalch — APPROVED** [review]

**Review by anastasiiakolomyi — CHANGES_REQUESTED** [review]
add body

**Review by anastasiiakolomyi — APPROVED** [review]
good

**Review by RamzesUAA — APPROVED** [review]
Good job.

**Review by anastasiiakolomyi — APPROVED** [review]
good

**Pull request merged**